mainnet/.env

```
@@ -1,21 +1,16 @@
-INSTANCE_NAME=XF_MasterNode
+INSTANCE_NAME=host4_XF_MasterNode
 CONTACT_DETAILS=YOUR_EMAIL_ADDRES
 SYNC_MODE=full
 GC_MODE=full
 NETWORK=mainnet
 PRIVATE_KEY=3333333333333333333333333333333333333333333333333333333333333333
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PRIVATE_KEY key should go before the SYNC_MODE key

(UnorderedKey)